lib/tag-list/input.tsx

```diff
@@ -1,49 +1,78 @@
-import React, { Component, FocusEvent, MouseEvent } from 'react';
+import React, {
+  ChangeEvent,
+  FocusEvent,
+  FunctionComponent,
+  useState,
+} from 'react';
```

Comment:
take this with a grain of salt since the code already exists before this PR but I find it convenient to reference the React events directly from the `React` object instead of importing them and referencing them in _unqualified_ ways. this is because the DOM platform already defines most of the events with the same name and it can be tricky to distinguish which one is in play or which one is supposed to be in play.

for example, with no import statement `FocusEvent` is already a defined type but it has different properties than `React.FocusEvent`

this has caused enough confusion for me that I have started using `FocusEvent` (and its friends) only when I mean "the DOM event" and `React.FocusEvent` when I am dealing with a React event.

Reply:
That makes a lot of sense. I updated this for both FocusEvent and also ChangeEvent which was done the same way.